src/web/services/http-request.service.ts

```diff
@@ -1,10 +1,35 @@
-import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
+import { HttpClient, HttpHeaders, HttpParameterCodec, HttpParams } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { environment } from '../environments/environment';
 import { ResourceEndpoints } from '../types/api-endpoints';
 import { MasqueradeModeService } from './masquerade-mode.service';
 
+/**
+ * This encoder was created to support encoding and decoding of plus (+) signs.
+ *
+ * Angular will ignore the encoding for plus signs. Refer to:
+ * https://github.com/angular/angular/blob/8.0.0/packages/common/http/src/params.ts#L33
+ */
+class CustomEncoder implements HttpParameterCodec {
```

Comment:
- Does it get fixed in Angular 9

- Can we extend `HttpUrlEncodingCodec` and only override `encodeValue`.

Reply:
I can confirm that the problem is still there. I will work on extending the codec.

Reply:
I just realised you're referring to a different codec than what was implemented. Are you suggesting an alternative implementation?